README.md

```diff
@@ -165,7 +172,7 @@ Change Log
 Documentation
 -------------
 
-The functionality of the specific classes in Open XML SDK Version 2.8.0 is similar to version 2.5, therefore the [Open XML SDK 2.5 for Office](http://msdn.microsoft.com/en-us/library/office/bb448854.aspx) documentation available on MSDN is still accurate.
+The functionality of the specific classes in Open XML SDK Version 2.8.1 is similar to version 2.5, therefore the [Open XML SDK 2.5 for Office](http://msdn.microsoft.com/en-us/library/office/bb448854.aspx) documentation available on MSDN is still accurate.
```

Comment:
It may be good to remove the reference to the latest version so we don't need to keep updating it

Reply:
Good idea. I suggest we change "... in Open XML SDK version 2.8.1 is..." to "... in this version of the Open XML SDK is...".

Reply:
I'll update